packages/arb-ts/src/lib/l2Bridge.ts

```diff
@@ -36,31 +36,27 @@ import {
   ARB_RETRYABLE_TX_ADDRESS,
 } from './precompile_addresses'
 import { Network } from './networks'
+import { ArbMulticall2__factory } from './abi/factories/ArbMulticall2__factory'
+import { MulticallFunctionInput, BridgeHelper } from './bridge_helpers'
 
 export interface L2TokenData {
```

Comment:
Same as above comment r.e. L1TokenData: should probably just get rid of the ERC/CUSTOM layer of the interface